OpenTreeMap/otm-core
# Remove triggers

Inserting a row into `treemap_mapfeature` on my development VM now takes 1 ms instead of 100 ms.

Importing 10,000 trees on my development VM now takes 8 minutes instead of 17 minutes. 
(Or, excluding verification time, 5 minutes instead of 14 minutes.)

Connects #2004 

To test:
* Verify that adding, moving, or deleting a tree is shown correctly on the map.
* Verify that the `INSERT` query in #2004 takes 1ms
